index.md

```diff
@@ -82,6 +82,8 @@ _For those thinking: “We don’t have anything yet, where do we start?”_
 * [Inclusion Ventures](http://inclusionventures.com/), a strategic consulting firm that provides training to help organizations and VCs take an inclusive approach to diversity and leadership
 * [FemaleFounder.org](https://www.femalefounder.org/), a community of women helping women (e.g., female founder office hours hosted across the country with top-tier female VCs)
 * [Parity.org](https://www.parity.org/), an organization that encourages corporations to commit to interviewing at least one qualified woman for all open roles at the VP level and above
+* [ReadySet](http://www.thereadyset.co/), is a strategy and solutions firm that helps innovative organizations become more diverse, equitable, and inclusive 
```

Comment:
1. To keep consistency with the other bullets, I'd make this not a sentence (i.e. take out the verb "is")
2. Still sounds a lot like marketing talk; maybe "a strategy and solutions firm that helps organizations become more diverse, equitable, and inclusive"?